### ISSUE TITLE
G0 - French titles cannot be given language markup

### ISSUE THREAD

**Issue by jpinsent** [ccswbs/hjckrrh#132]
The title field cannot be given a span attribute. Pages entirely in French can only have their body content marked as French.

**Comment by mmafe** [comment]
In order to fix, we would need to ensure that wherever the title appears on the site (e.g. teaser lists, listing pages, node detail page) it has a language attribute.

Unfortunately, this cannot be done quickly nor easily, so I'm taking the issue out of the hotfix milestone and removing the high priority until further planning has been done for multilingual support.

**Comment by abanuog** [comment]
Have an english title and in the first line of the body, state that the french text follows the english text.
